packages/test/test-end-to-end-tests/src/test/gc/gcSweepDataStores.spec.ts

```diff
@@ -541,23 +541,245 @@ describeCompat("GC data store sweep tests", "NoCompat", (getTestObjectProvider)
 		});
 
 		it("can run sweep without failing summaries due to local changes", async () => {
-			const { summarizer } =
-				await summarizationWithUnreferencedDataStoreAfterTime(sweepTimeoutMs);
-			await sendOpToUpdateSummaryTimestampToNow(summarizer);
+			const { summarizer } = await summarizationWithUnreferencedDataStoreAfterTime();
 
 			// Summarize. In this summary, the gc op will be sent with the deleted data store id. Validate that
 			// the GC op does not fail summary due to local changes.
 			await assert.doesNotReject(
-				async () => summarizeNow(summarizer),
+				async () => summarize(summarizer),
 				"Summary and GC should succeed in presence of GC op",
 			);
 
 			// Summarize again so that the sweep ready blobs are now deleted from the GC data. Validate that
 			// summarize and GC succeed.
 			await assert.doesNotReject(
-				async () => summarizeNow(summarizer),
+				async () => summarize(summarizer),
 				"Summary and GC should succeed with deleted data store",
 			);
 		});
 	});
+
+	describe("Sweep with summarize failures and retries", () => {
```

Comment:
Since the scenario we're trying to resolve is creating a datastore on the first summarize run, could we add a test to do that here so that we can see if we fix that scenario?

Reply:
That's not what these tests are for. They are simply testing that if multiple GC ops are sent with the same deleted node ids, things still work as expected. It doesn't care about what the failure is that leads to it.